composer/loggers/file_logger.py

```diff
@@ -43,6 +43,15 @@ class FileLogger(LoggerCallback):
                 loggers=[logger]
             )
 
+        .. testcleanup::
```

Comment:
The file logger test was writing a file that never got removed so when you ran `make doctest` multiple times it failed saying the file already exists

Reply:
damn good catch